src/handlers.js

```diff
@@ -48,8 +48,9 @@ exports.generateWrapperExt = function(func) {
  * @param {Object} func The function to wrap.
  * @return {String} The wrapper code.
  */
-exports.generateWrapperCode = function(func) {
-    let customNodePath = _.get(func, 'custom.thundra.node_modules_path', '')
+exports.generateWrapperCode = function(func, config) {
+    let customNodePath =
+        func.node_modules_path || config.node_modules_path || ''
```

Comment:
I think, `func.node_modules_path` doesn't get `node_modules_path` config defined with `custom.thundra.` prefix under function. I mean this one:
```
functions:
  hello-world-test:
    name: hello-world-test
    handler: index.handler
    custom:
      thundra:
        node_modules_path: <directory that contains @thundra/core>
```

So what about 
```
let customNodePath =
    _.get(func, 'custom.thundra.node_modules_path') || config.node_modules_path || ''
```

Reply:
Ah I misread that part in the README (thought it was added globally). Will fix as suggested :) 

So yeah I'm actually adding a new feature as well: being able to define node_modules_path globally :D